src/main/java/org/apache/mesos/config/DefaultConfigurationValidator.java

```diff
@@ -0,0 +1,27 @@
+package org.apache.mesos.config;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * Default Configuration validator, which validates a given new {@code Configuration} w.r.t.
+ * an old {@code Configuration}.
+ */
+public class DefaultConfigurationValidator {
+    private final Collection<ConfigurationValidation> validations = new ArrayList<>();
+
+    public DefaultConfigurationValidator(final Collection<ConfigurationValidation> validations) {
```

Comment:
Would be useful to have an alternate constructor of the form:

```
Constructor(ConfigurationValidator... validators) {
  this(Arrays.asList(validators));
}
```

Then callers could create a new validator like this:

```
new DefaultConfigurationValidator(
  new MyValidatorA(),
  new MyValidatorB(),
  ...);
```